README.md

```diff
@@ -86,10 +88,136 @@ transformer.renderFileAsync(filename[, options], locals);
 => Promise({body: String, dependencies: Array.<String>})
 ```
 
-_requires the underlying transform to implement `.renderFileAsync`, `.renderFile`, `.renderAsync`, `.render`, `.compileFileAsync`, `.compileFile`, `.compileAsync`, or `.compileFile`_
+_requires the underlying transform to implement `.renderFileAsync`, `.renderFile`, `.renderAsync`, `.render`, `.compileFileAsync`, `.compileFile`, `.compileAsync`, or `.compile`_
 
 Transform a file asynchronously. If a callback is provided, it is called as `callback(err, data)`, otherwise a Promise is returned.
 
+### `.compile*`
+
+#### Returned object from `.compile*`
+
+```js
+{fn: Function, dependencies: Array.<String>}
+```
+
+ - `fn` represents the result as a string
```

Comment:
fn is a function that takes locals and returns a string.  It is not a string itself.